support-frontend/assets/helpers/urls/routes.ts

```diff
@@ -110,13 +110,20 @@ function guardianWeeklyLanding(countryGroupId: CountryGroupId, gift: boolean) {
 const promotionTermsUrl = (promoCode: string) =>
 	`${getOrigin()}/p/${promoCode}/terms`;
 
+function getPaperOrigin(productOptions: ProductOptions): string {
+	const origin = getOrigin();
+	return productOptions === 'Sunday'
+		? origin.replace('support', 'observer')
```

Comment:
I wonder if this could be more specific? Right now I think this will replace the string `support` anywhere in the origin, maybe that's fine, but could be more exact? Maybe that'd make the intention clearer? Like this maybe? It's also be good to write a test around this if possible!

```suggestion
		? origin.replace(/^https:\/\/support/, "https://observer")
```

What do you reckon? It's definitely possible I'm overthinking this 😄

Reply:
Yep, good to be more specific, have also added the tests  for Observer and Guardian, Updated.